lib/user.rb

```diff
@@ -0,0 +1,24 @@
+require 'active_record'
+
+class User < ActiveRecord::Base
+
+  has_many :peeps
+
+  validates :email, confirmation: true
+  validates :email, :uniqueness => { :message => "This email already exists" }
+  validates :username, :uniqueness => 
+  { :message => "This username already exists" }
```

Comment:
Do these messages actually display anywhere? I can't seem to find them in the code